trade-finance-manager-ui/styles/table.scss

```diff
@@ -156,4 +156,26 @@ table.premium-payments-table {
   .payments-list-item {
     list-style-type: none;
   }
-}
\ No newline at end of file
+}
+
+table.utilisation-report-reconciliation-table {
+  table-layout: fixed;
+
+  .ukef-width-one-tenth {
+    width: 10% !important;
+  }
+
+  .ukef-width-one-quarter {
+    width: 25% !important;
+  }
+}
+@media (max-width:65em) {
+  table.utilisation-report-reconciliation-table {
+    width: 55em;
+
+    .govuk-table__header,
+    .govuk-table__cell {
+      white-space: normal;
+    }
+  }
+}
```

Comment:
Since we're using SCSS we can save some lines here by nesting the media query in the selector like this:

```css
table.utilisation-report-reconciliation-table {
  table-layout: fixed;

  /* ... */

  @media (max-width:65em) {
    width: 55em;

    .govuk-table__header,
    .govuk-table__cell {
      white-space: normal;
    }
  }
}
```

Reply:
Cool, I wasn't sure which to adopt due to `layout.scss` using the other way. Changed this now in line with your suggestion thanks